### DISCUSSION TITLE
Pick and place file for Light Unit 2

### DISCUSSION THREAD

**Discussion by vandergus** [srozum/film_camera_tester#35] [Q&A] [answered]
Is there a pick and place file available for Light Unit 2? I see them for other PCBs but not for this one. It's not a big deal to hand assemble but it would be nice if I could get JLC PCB to do it for me. Cheers.

https://github.com/srozum/film_camera_tester/tree/master/assets/pcb/light_unit_2

**Comment by srozum** [comment, marked as answer]
Because pre-build LED driver is used in LU design the rest of components are through the hole. And JLC PCB doesn't offer through the hole assembly.

**Reply by vandergus** [reply]
Gotcha, thanks for the quick reply.
